.github/workflows/soak.yaml

```diff
@@ -208,6 +210,7 @@ jobs:
                 echo "$pod: ${pod_map[$pod]}"
                 if [ "${pod_map[$pod]}" -gt 1 ]; then
                   echo "Test failed: Pod $pod has count ${pod_map[$pod]}"
+                  kubectl logs deploy/pepr-soak-ci-watcher -n pepr-system
```

Comment:
May be worth pushing this log dump into the `watch-logs.txt` file too, no?  Not sure that the bash logs get truncated like our JS-based-test log output (they don't, right?), but it'd be nice to have a version of it that can be downloaded & scripted through in case the output get's really large.

Or maybe it's not worth the trouble?

Reply:
Problem is the test exits and does not dump

https://github.com/defenseunicorns/pepr/actions/runs/11147804732/job/30983047281

Reply:
so this is like the last mile to throw it in the logs